public/index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="x-ua-compatible" content="ie=edge" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
+    <title data-react-helmet="true"></title>
+    <script src="/socket.io/socket.io.js"></script>
+  </head>
+  <body>
+    <div id="___gatsby"></div>
+    <script src="/commons.js"></script>
+  </body>
+</html>
```

Comment:
What's this file for?

Reply:
don't know... Did I push that? it must be an auto-generated file

Reply:
I don’t think so, it says Gatsby, and you don’t creste an `index.html` with next